tfrobot/example/test.yaml

```diff
@@ -16,6 +16,11 @@ node_groups:
     free_cpu: 2
     free_mru: 8
 
+  - name: group_d
+    nodes_count: 2
+    free_cpu: 2
+    free_mru: 16
```

Comment:
FYI: you can request less mru because you are using only 2 vms with 1 memroy (cpu too)